crates/misc/dotnet/Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
-    <WasmtimeVersion>0.8.0</WasmtimeVersion>    
+    <WasmtimeVersion>0.8.0</WasmtimeVersion>
```

Comment:
cc @peterhuene, mind peeking at this and the next file to ensure that this is still at least somewhat idiomatic?

Reply:
LGTM 👍